### ISSUE TITLE
[[ not found

### ISSUE THREAD

**Issue by jsanz** [elastic/kibana-demo-data#10]
Run the helper with the suggested script interpreter `sh` and works great 🥳 but eventually got into a loop of `[[ not found` which makes sense because `[[ ... ]]` is not valid in `sh` so I think the usage of `[[` needs to be reviewed or suggest using `bash` in the overall script launching.

```
[2024-09-30T18:46:07+0200][all] - Executing all scripts
Waiting for Elasticsearch to be online...
..Elasticsearch is online.
Installing custom sample data to Elasticsearch
Processing log-apache_error
Processing log-apache_error completed
Processing log-aws_s3
Processing log-aws_s3 completed
Processing log-custom_multiplex
Processing log-custom_multiplex completed
Processing log-k8s_container
Processing log-k8s_container completed
Processing log-nginx_error
Processing log-nginx_error completed
Processing log-nqinx
Processing log-nqinx completed
Processing log-system_error
Processing log-system_error completed
Processing custom-metrics-without-timestamp
Processing custom-metrics-without-timestamp completed
Installing custom sample data to Elasticsearch finished
Waiting for Kibana to be online...
sh: 87: [[: not found
.sh: 87: [[: not found
.sh: 87: [[: not found
.sh: 87: [[: not found
.sh: 87: [[: not found
.sh: 87: [[: not found
.sh: 87: [[: not found
...
```

I found `[[` here:

https://github.com/elastic/kibana-demo-data/blob/1c4fd02ef496be87ad64276a4896cd6a262a4170/scripts/custom.sh#L87

Hope it helps!

**Comment by kertal** [comment]
Sure! Thx, just testing if it works with 

```
if [[ -z "$dev_prefix" ]]; then
```

**Comment by kertal** [comment]
Should be fixed with 2142a1a8cb0c3d8aa2e1323cfd2a823096216a94 , thx @jsanz
